metabase/metabase
# Polish appearance settings

Epic: #38699
Target release: 0.50

### Description

Changes in this PR:
1. Add tabs to appearance settings and put each tab on its own route
1. Move settings to the new *Conceal Metabase* tab
1. Order settings to match the design.
1. Change Favicon widget to file upload like the logo setting.

### How to verify
1. Go to admin settings > Appearance > there would be 2 tabs
2. You can click between the 2 tabs and each of them should have its own URL. Try to refresh and see that if users were on the 2nd tab then it would show the 2nd tab after the refresh.
3. Check that setting the application names doesn't change charts' colors
4. Check that on the general tab we can see the landing page only if you have a valid license.

### Checklist
